src/agent/src/agent.cpp

```diff
@@ -10,11 +9,14 @@ Agent::Agent()
     m_taskManager.start(std::thread::hardware_concurrency());
     m_taskManager.enqueueTask(StatefulMessageProcessingTask(m_messageQueue));
     m_taskManager.enqueueTask(StatelessMessageProcessingTask(m_messageQueue));
+
+    int authRes = m_communicator.SendAuthenticationRequest();
+    int regRes = m_communicator.SendRegistrationRequest();
 }
 
 Agent::~Agent()
 {
     // Sleep for 2 seconds
     std::this_thread::sleep_for(std::chrono::seconds(2));
     m_taskManager.stop();
-}
+}
```

Comment:
Missing new line at the end of the file.

Reply:
Done